google-cloud-tasks/samples/create_http_task_with_token.rb

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloud_tasks_create_http_task_with_token]
+require "google/cloud/tasks"
+
+# Create a Task with an HTTP Target With Token
```

Comment:
Maybe instead of "With Token" ("With" should not be capitalized):

    Create a Task with an HTTP Target and OIDC token